src/archanaNJun20/SumOfNumbers.java

```diff
@@ -0,0 +1,23 @@
+package archanaNJun20;
+public class SumOfNumbers{
+	void printSumOfDigits(String input){
+		int sum = 0;
+		String[] array= input.split(" ");
+		for (int index = 0; index < array.length;){
+			try{
+				sum += Integer.parseInt(array[index]);
+			} 
+			catch (NumberFormatException e){ 
+			}
+			finally {
+				index++;
```

Comment:
Archana finally block is not required because your exception is getting catch by catch block. So your program will not terminate. So is it okay if you do increment using for loop.